src/order/filter/ultrafilter.lean

```diff
@@ -313,3 +313,37 @@ compl_compl s ▸ hf.compl_mem_hyperfilter
 end hyperfilter
 
 end filter
+
+namespace ultrafilter
+
+lemma eq_of_le {f g : ultrafilter α} (h : (f : filter α) ≤ g) : f = g :=
+coe_injective (g.unique h)
```

Comment:
This is just one direction of `coe_le_coe` on line 53, right? Can you change `coe_le_coe` to be proved in terms of this lemma?

Reply:
Done.